### PR TITLE
[REF] consider-merging-classes-inherited: Skip check if there is '_name' attribute

### PR DESCRIPTION
**Pull request by tomeyro**
Referring issue: https://github.com/OCA/pylint-odoo/issues/92

### CODE REVIEW

File: pylint_odoo/checkers/modules_odoo.py [hunk 1]
```diff
@@ -260,7 +260,10 @@ def visit_assign(self, node):
         if node_left.name == '_name':
             node.parent.odoo_attribute_name = node.value.value
             return
-        _name = getattr(node.parent, 'odoo_attribute_name', None)
+        try:
+            _name = node.parent.igetattr('_name').next().value
+        except:
```
Comment (moylop260) [inline pylint_odoo/checkers/modules_odoo.py]: too bare exception, use custom exception

Comment (tomeyro) [inline pylint_odoo/checkers/modules_odoo.py]: done

**Comment by moylop260** [comment]
Could you run the lints for [addons-vauxoo#8.0](https://github.com/vauxoo/addons-vauxoo/tree/8.0)?
`~/pylint-odoo$  pylint -d all -e consider-merging-classes-inherited --load-plugins=pylint_odoo ~/addons-vauxoo`
And check if we don't have other exception not cached.

**Comment by tomeyro** [comment]
Is this the expected output?

captura de pantalla de 2016-12-23 12-36-40

**Comment by moylop260** [comment]
Sorry I missed comment you that is required create a `__init__.py` in the root of addons-path.
Run `touch ~/addons-vauxoo/__init__.py` and repeat the last command, please

**Comment by tomeyro** [comment]
@moylop260 output:
image

**Pull request merged**